open-source-software-peer-review/intro.md

```diff
@@ -1,13 +1,127 @@
+"peer-reviewed" badge at the top their README, [![pyOpenSci](https://tinyurl.com/y22nb8up)]() linking to the specific issue
+where the tool was reviewed. [See this example from devicely, one of our packages](https://github.com/hpi-dhc/devicely).
 
+### How do reviews work?
 
+We use GitHub for our entire review process. We like GitHub because:
 
+* It's free to create an account
+* It facilitates collaboration and supports community around a package
+* It facilitates open discussion via issues
+* It supports version control
+* Numerous packages store their code bases on GitHub
+
+We make the most of [GitHub](https://github.com/)infrastructure in
+our review process.
+
+Each package review is contained within an issue in the [pyOpenSci/software-review GitHub repository](https://github.com/pyopensci/software-review/). 
+
+# TODO: ADD GOOD EXAMPLE HERE
+For instance, click [here](https://github.com/ropensci/software-review/issues/24) to read the review thread of the `ropenaq` package: the process is an ongoing conversation until acceptance of the package, with two external reviews as important milestones. 
+
+### GitHub tools including issue submission templates and labels help us streamline peer review
+We use GitHub features including:
+
+* issue templates (as submission templates), and
+* labelling issues to track progress of submissions (from editor checks to approval).
+* Project boards to track help wanted items
+
+All of this functionality supports a streamlined and open peer review
+process. 
+
+## Why review a package for pyOpenSci?
+
+We hope you choose to review **to give back to the rOpenSci and scientific communities.**  Our mission to expand the ability to efficiently use scientific data and promote a culture of open reproducible is only possible through the volunteer efforts of community members like you.
+
+-   Peer review is a two-way conversation. By reviewing packages, you'll have the chance to **continue to learn development practices from authors and other reviewers**.
+-   The open nature of our review process allows you to **network and meet colleagues and collaborators** through the review process. Our community is friendly and filled with supportive members expert in Python package development, a diversity of science domains and computer science.
+
+### Volunteer to review for us
+-   To volunteer to be one of our reviewers, fill out [this short form](https://forms.gle/9dv99gri6XKNU8177). In the form you will provide your contact information and areas of expertise. We are always looking for more reviewers with both general package-writing experience and domain expertise in the fields where packages are used. Some of our reviewers focus on package usability and documentation. This allows you to review even if you don't have strong technical background. We will pair you with a second reviewer that can focus more on the technical aspect of the review. 
```

Comment:
I think review for us should be in a few places potentially even if it's a link to the website where we have the form.